api/api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayAggregations.scala

```diff
@@ -48,7 +48,9 @@ object DisplayAggregations {
       workType = displayAggregation(aggs.workType, DisplayWorkType.apply),
       productionDates =
         displayAggregation(aggs.productionDates, DisplayPeriod.apply),
-      genre = None
+      genres = displayAggregation(
+        aggs.genres,
+        (g: Genre[Displayable[AbstractConcept]]) => DisplayGenre(g, false))
```

Comment:
If `includesIdentifiers` default to `false`, then could just put `DisplayGenre.apply` here? Which would make it a bit easier to read imo

Reply:
Yea, I was trying not to let the somewhat strange code of the aggregations meddle with code elsewhere.

I wondered if not having the default in the case class made the code more explicit around the code base.

By setting the default to false, it feels like we're expressing that they dont normally have them, and on special situations they do. 

So I didn't want to make that implication for the sake of readability in 1 place. 

But I've probably completely over thought that.

Reply:
In that case, this potentially more readable, but just as bad??

```scala
genres = displayAggregation[Genre[Displayable[AbstractConcept]], DisplayGenre](
  aggs.genres,
  DisplayGenre(_, false)
)
```